README.md

```diff
@@ -28,8 +28,22 @@ fmt.Println(lang)
 lang, _ = GetLanguageByContent("bar.m", "<pbjective-c-code>")
```

Comment:
Sorry for raising a point out of the context of your PR but I just notice an errata; do you mind fix it, please?: s/pbjective/objective/